tests/Fixer/LanguageConstruct/CombineConsecutiveIssetsFixerTest.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\LanguageConstruct;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author SpacePossum
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\LanguageConstruct\CombineConsecutiveIssetsFixer
+ */
+final class CombineConsecutiveIssetsFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideCases()
+    {
+        return [
+            [
+                '<?php $a = isset($a, $b)  ;',
+                '<?php $a = isset($a) && isset($b);',
+            ],
+            [
+                '<?php $a = isset($a, $b,$c)  ;',
+                '<?php $a = isset($a) && isset($b,$c);',
+            ],
+            [
+                '<?php $a = isset($a,$c, $b,$c)  ;',
+                '<?php $a = isset($a,$c) && isset($b,$c);',
+            ],
+            [
+                '<?php $a = isset($a,$c, $b)  ;',
+                '<?php $a = isset($a,$c) && isset($b);',
+            ],
+            [
+                '<?php $a = isset($a, $b)   || isset($c, $e)  ?>',
+                '<?php $a = isset($a) && isset($b) || isset($c) && isset($e)?>',
+            ],
+            [
+                '<?php $a = isset($a[a() ? b() : $d], $b)  ;',
+                '<?php $a = isset($a[a() ? b() : $d]) && isset($b);',
+            ],
+            [
+                '<?php $a = isset($a[$b], $b/**/)  ;',
+                '<?php $a = isset($a[$b]/**/) && isset($b);',
+            ],
+            [
+                '<?php $a = isset ( $a, $c, $d /*1*/ )                 ;',
+                '<?php $a = isset ( $a /*1*/ )    &&    isset ( $c   ) && isset( $d );',
+            ],
+            'minimal fix case' => [
+                '<?php isset($a, $b);',
+                '<?php isset($a)&&isset($b);',
+            ],
+            [
+                '<?php isset($a, $b, $c)    ;',
+                '<?php isset($a) && isset($b) && isset($c);',
+            ],
+            [
+                '<?php $a = isset($a,$c, $b,$c, $b,$c,$d,$f, $b)      ;',
+                '<?php $a = isset($a,$c) && isset($b,$c) && isset($b,$c,$d,$f) && isset($b);',
+            ],
+            'comments' => [
+'<?php
+
+$a =#0
+isset#1
+(#2
+$a, $b,$c, $d#3
```

Comment:
probably this fixer needs to be executed before "space after comma" fixer as well

Reply:
None of the `comma` fixers touch these cases, because they either deal with methods or array's.
I could've overseen one of course so happy to test more if you know one.

Reply:
we don't fix commas in functions ?

Reply:
we do:
`php-cs-fixer describe method_argument_space`, example `#6`:
```diff
+++ New
@@ @@
<?php
-function sample($a=10,
-    $b=20,$c=30) {}
-sample(1,  
-    2);
-sample('foo',    'foobarbaz', 'baz');
-sample('foobar', 'bar',       'baz');
+function sample(
+    $a=10,
+    $b=20,
+    $c=30
+) {}
+sample(
+    1,
+    2
+);
+sample('foo', 'foobarbaz', 'baz');
+sample('foobar', 'bar', 'baz');
```

Reply:
```
$ php php-cs-fixer fix test2.php --dry-run --diff -vvv
Loaded config default from "/home/possum/work/PHP-CS-Fixer/.php_cs.dist".
```

```diff
--- Original
+++ New
@@ @@

-$a = isset($a,   $b  ,    $a)   ;
+$a = isset($a,   $b  ,    $a);
```

My guess is that the white space within the braces is not fixed because `isset` is a language construct and not a function. 
Secondly this fixer will only an single space within the `isset` braces if there is no space at all, making the order in which to such white space is fixed by others fixer not important (as we have non that would remove the added whitespace).